npa.js

```diff
@@ -17,6 +17,7 @@ let os
 const isWindows = process.platform === 'win32' || global.FAKE_WINDOWS
 const hasSlashes = isWindows ? /\\|[/]/ : /[/]/
 const isURL = /^(?:git[+])?[a-z]+:/i
+const isGit = /^[^@]+@[^:.]+\.[^:]+:/i
```

Comment:
I'd put a `.+$` at the end of this, just to be sure that something comes after the `:`.